app/models/requests/AuthenticatedRequest.scala

```diff
@@ -19,6 +19,6 @@ package models.requests
 import models.SignedInUser
 import play.api.mvc.{Request, WrappedRequest}
 
-class AuthenticatedRequest[A](request: Request[A], val user: SignedInUser) extends WrappedRequest[A](request) {
+class AuthenticatedRequest[+A](request: Request[A], val user: SignedInUser) extends WrappedRequest[A](request) {
```

Comment:
Is this necessary? How we need it now?

Reply:
Technically wrappers around request should have same variance as orginal Request so it should be covariant